parquet-tools/README.md

```diff
@@ -99,9 +99,10 @@ where option is one of:
     -c,--column <arg>  Dump only the given column, can be specified more than
                        once
     -d,--disable-data  Do not dump column data
-       --debug         Disable color output even if supported
+       --debug         Enable debug output
```

Comment:
Although do we even need the whole help string in the README? This could happen again if the options change. 

Might be better to just direct the user to run `-h` to get usage details